terraform/modules/services/api-deployment/main.tf

```diff
@@ -17,6 +17,10 @@ resource "aws_api_gateway_deployment" "fat" {
     var.guided_match_api_gateway_integration
   ]
 
+  triggers = {
+    redeployment = sha1(var.scale_rest_api_policy_json)
+  }
```

Comment:
So this triggers a redeployment on policy change I guess?

Reply:
Yep exactly - uncovered it during the first round of changes to make the APIs private (which also involved a policy). This seemed the most reliable way to initiate a deployment.  Otherwise you are forced to make a manual deployment, but then when you run the next `terraform apply` it sees drift in the deployment ID and overwrites it back to _not having_ a policy at all, or the previous version.